src/platform/forms-system/src/js/utilities/save-in-progress-messages.js

```diff
@@ -0,0 +1,19 @@
+import { APP_TYPE_DEFAULT } from '../constants';
+
+const appType = formConfig =>
+  formConfig.customText?.appType || APP_TYPE_DEFAULT;
+
+export function inProgressMessage(formConfig) {
```

Comment:
This doesn't _really_ matter, but would it make more sense if this file were inside `src/platform/forms/save-in-progress`?

Reply:
Yeah, maybe. Since we planned to consolidate these two directories anyhow, I figured it didn't matter much. :man_shrugging: 

I don't have a strong opinion. I can move it, if you think that'd be better.